operator-framework-core/src/main/java/io/javaoperatorsdk/operator/utils/CounterDetails.java

```diff
@@ -0,0 +1,40 @@
+package io.javaoperatorsdk.operator.utils;
+
+public class CounterDetails {
+
+    int counter;
```

Comment:
do these fields need to be package-private instead of private?

Reply:
checking on this, we can make those field as a private.

Reply:
I don't currently see what's the point of this class since it doesn't appear to be used but I would assume that it should be immutable: all fields should be private and final and there shouldn't be any setters (i.e. it shouldn't be possible to modify an instance, after it's been created)

Reply:
The counter details class will be used when we call/use these utility methods. You can refer to sample **memcached-quarkus-operator** where I used this - https://github.com/laxmikantbpandhare/memcached-quarkus-operator/blob/main/src/main/java/com/example/MemcachedController.java